src/__tests__/components/frontend-engine/frontend-engine.spec.tsx

```diff
@@ -188,6 +188,34 @@ describe("frontend-engine", () => {
 		expect(getErrorMessage(true)).not.toBeInTheDocument();
 	});
 
+	describe.only("setErrors", () => {
+		const handleClick = async (ref: React.MutableRefObject<IFrontendEngineRef>) => {
+			try {
+				throw new Error("API error");
+			} catch (error) {
+				ref.current.setErrors({
+					[fieldOneId]: ERROR_MESSAGE,
+				});
+			}
+		};
+
+		it("should support setting of custom errors", async () => {
```

Comment:
as `setErrors()` accepts error message as multiple types like string and array, there is a need to assert the support for each type